osf_tests/test_files.py

```diff
@@ -51,3 +52,16 @@ def test_active_manager_does_not_return_trashed_file_nodes(project, create_test_
     assert BaseFileNode.objects.filter(node=project).count() == 3
     # root folder + file = 2 BaseFileNodes
     assert BaseFileNode.active.filter(node=project).count() == 2
+
+def test_folder_update_calls_folder_update_method(project, create_test_file, monkeypatch):
```

Comment:
I don't disagree. What I really wanted to do was assert that the method on the child class was the same as the method on the Folder class. I couldn't find a way to do that. The suggested alternative on "The Internet" was to mock the method to do something obvious and simple to check and then call it like normal and assert the return was as expected. 

The test is basically to make sure that the Folder.update method continues to exist and that the Folder subclasses in addons, like OsfStorageFolder.update, continue to call Folder.update.

All that said, if you've got a better way to test that the method I'm trying to call is the method I'm trying to call I'm happy to change the implementation. Or I can put a comment in the test explaining why it seems a bit off.